client/src/services/categories.service.ts

```diff
@@ -1,7 +1,25 @@
 import axiosInstance from './axios.service';
+import { store } from "../store/store";
+import { Gender } from "../store/auth/auth.model";
+
+export interface Category {
```

Comment:
After merge conflicts, remove this because it'll be defined in a separate model file